repos/system_upgrade/el7toel8/libraries/dnfplugin.py

```diff
@@ -89,7 +89,7 @@ def backup_debug_data(context):
             api.current_logger().warn('Failed to copy debugdata. Message: {}'.format(e.message), exc_info=True)
 
 
-def _transaction(context, stage, target_repoids, tasks, test=False):
+def _transaction(context, stage, target_repoids, tasks, cmd_env_vars=None, test=False):
```

Comment:
Why not just env or at least cmd_env

Reply:
Fixed the namings. It was indeed too long.